pages/infrastructure-as-code-library/_library.html

```diff
@@ -5,7 +5,7 @@ <h2>What's in the Library?</h2>
   infrastructure code for AWS, written in Terraform, Go, Bash,
   and Python. Check out
   <a
-    href="/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library/"
```

Comment:
IIRC, this guide is linked to from a bunch of other places: e.g., module READMEs. So fixing it here alone won't be enough. We really need to ensure we have redirects in place!